app/controllers/events_controller.rb

```diff
@@ -31,13 +31,13 @@ def create
     @users = User.all
     authorize @event
     if @event.save
-      @timetable = Task.new(name: "Timetable", description: "Set times for this event", category: "music", start: @event.start_date, end: @event.end_date, timetable: true)
```

Comment:
Why was the start time removed from these?
I think this might be part of why we're getting an error.
It's expecting a start time in the Timetable render.